.vib/grafana/vib-pipeline.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
this is not needed, the 'early-feedback' action already covers 'package', 'lint' and 'trivy' for all the Helm charts.

Reply:
Removed.